src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const peoples = [];
+
+for (let i = 0; i < population.length; i++) {
+  if (!isNaN(Number(population[i].textContent.split(',').join('')))) {
```

Comment:
would be better to create a variable for this value. We don't need to calculate it twice
`const countryPopulation = Number(population[i].textContent.split(',').join(''));`